### DISCUSSION TITLE
Snapshot and RTMP are using the low-resolution stream

### DISCUSSION THREAD

**Discussion by veno77** [blakeblackshear/frigate#1829] [General]
Hello, I am using Foscam 1080P cameras. Here is my config:

  doorway_frigate:
    ffmpeg:
      inputs:
        - path:  rtsp://user:pass@192.168.100.91:91/videoMain
          roles:
            - rtmp
            - clips
        - path:  rtsp://user:pass@192.168.100.91:91/videoSub
          roles:
            - detect
    width: 1280
    height: 720
    fps: 5
    objects:
      track:
        - person
        - car
      filters:
        car:
          mask:
            - 790,235,1086,464,758,720,237,440
    motion:
      mask:
        - 327,0,319,161,165,243,138,125,0,85,0,0,114,0
        - 1160,61,1185,97,1180,147,1151,175,1045,177,1035,63
        - 0,720,327,720,0,210

The recorded clips are ok with 1080p quality, but the Snapshots and live view is with 720p. I want only the detect to use the low resolution stream for lowering the cpu usage. I thought that when I define clips and rtmp under the main stream, all live, clips and snapshots will use the stream under which they are defined?

**Comment by blakeblackshear** [comment]
Snapshots and live view come from the detect stream because it is the only stream that is decoded. If you want those from the higher resolution stream, you have to use more CPU to decode the frames so snapshots can be grabbed.
